lib/knapsack_pro/base_allocator_builder.rb

```diff
@@ -22,10 +22,15 @@ def test_dir
     end
 
     def test_files
-      found_test_files = KnapsackPro::TestFileFinder.call(test_file_pattern)
+      test_file_paths = KnapsackPro::TestFileFinder.call(test_file_pattern)
 
       if adapter_class == KnapsackPro::Adapters::RSpecAdapter && KnapsackPro::Config::Env.rspec_split_by_test_examples?
-        test_files_count = found_test_files.size
+        if slow_test_file_pattern
+          slow_test_file_paths = KnapsackPro::TestFileFinder.call(slow_test_file_pattern, test_file_list_enabled: false)
```

Comment:
TODO: Read list of slow test files fetched from API based on existing file on the disk created by lib/knapsack_pro/test_case_detectors/rspec_test_example_detector.rb